_rules/image-has-name-23a2a8.md

```diff
@@ -53,55 +53,64 @@ There is a known combination of a popular browser and assistive technology that
 
 #### Passed Example 1
 
-The HTML `img` element has an accessible name
+The HTML `img` element has an accessible name given by the `alt` attribute.
 
 ```html
 <img alt="W3C logo" />
 ```
 
 #### Passed Example 2
 
-The element with role of `img` has an accessible name
+The element with a [semantic role](#semantic-role) of `img` has an accessible name given by the `aria-label` attribute.
 
 ```html
 <div role="img" aria-label="W3C logo"></div>
 ```
 
 #### Passed Example 3
 
-The element has an accessible name, though the name is not always accessibility supported
+The element with a [semantic role](#semantic-role) of `img` has an accessible name given by an `aria-labelledby` attribute and element.
 
 ```html
-<img title="W3C logo" />
+<div style="display: hidden" id="img-label">W3C logo</div>
+<div role="img" aria-labelledby="img-label"></div>
 ```
 
 #### Passed Example 4
 
-The HTML `img` element is marked as [decorative](#decorative) through an empty `alt` attribute
+The HTML `img` element has an accessible name, though the name is not always accessibility supported.
```

Comment:
Just a suggestion: to be consistent with your other descriptions add the "given by the `title` attribute." information ?

And maybe replace "the name" by "the `title`" ? 


> The HTML `img` element has an accessible name given by a `title` attribute, though the `title` is not always accessibility supported.